lib/pack/localpack/packageserver.go

```diff
@@ -383,8 +384,12 @@ func (p *PackageServer) UnpackedPath(loc loc.Locator) (string, error) {
 }
 
 func (p *PackageServer) Unpack(loc loc.Locator, targetDir string) error {
-	var err error
+	return p.UnpackWithOptions(loc, targetDir, nil)
+}
 
+// UnpackWithOptions unpacks the package specified with loc in targetDir
+// using opts for extraction
+func (p *PackageServer) UnpackWithOptions(loc loc.Locator, targetDir string, opts *dockerarchive.TarOptions) (err error) {
```

Comment:
What is this new function for? It doesn't seem used except by the old `Unpack` -- in which case why add a new public function?

Reply:
Looks like this change shouldbe included in https://github.com/gravitational/gravity/pull/2498 instead -- that PR has the rest of the context.

Reply:
Removed.